src/librustc_llvm/build.rs

```diff
@@ -28,6 +28,11 @@ fn detect_llvm_link() -> (&'static str, &'static str) {
 }
 
 fn main() {
+    if env::var_os("RUST_CHECK").is_some() {
+        // If we're just running `check`, there's no need for LLVM to be built.
```

Comment:
Could this print out `rerun-if-env-changed` to automatically detect when this is altered?